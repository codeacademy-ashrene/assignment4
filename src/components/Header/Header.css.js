import { StyleSheet } from 'react-native';

export default StyleSheet.create({

  header: {
    flex: 1,
    flexDirection: 'row',
    justifyContent: 'flex-start',
  },
  text: {
    backgroundColor: '#3F51B5',
    color: '#FFFFFF',
    height: 80,
    fontSize: 30,
    width: '100%',
    padding: '5%',
  },
});
