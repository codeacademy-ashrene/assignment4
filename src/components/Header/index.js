import React, { Component } from 'react';
import { View, Text } from 'react-native';
import styles from './Header.css';

class Header extends Component {
  render() {
    return (
      <View style={styles.header}>
        <Text style={styles.text}>Awesome Forms</Text>
      </View>
    );
  }
}

export default Header;
