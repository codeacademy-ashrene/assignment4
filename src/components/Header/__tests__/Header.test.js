import React from 'react';
import renderer from 'react-test-renderer';
import Header from '..';


describe('Header', () => {
  it('should render Header', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
