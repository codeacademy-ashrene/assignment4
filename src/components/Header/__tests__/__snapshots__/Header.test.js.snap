// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Header 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "flex-start",
    }
  }
>
  <Text
    style={
      Object {
        "backgroundColor": "#3F51B5",
        "color": "#FFFFFF",
        "fontSize": 30,
        "height": 80,
        "padding": "5%",
        "width": "100%",
      }
    }
  >
    Awesome Forms
  </Text>
</View>
`;
